generators/pkg/writer/write.go

```diff
@@ -0,0 +1,41 @@
+// Package writer provides a method to write the generated task
+// to the yaml document.
+package writer
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"generators/pkg/generator"
+	"generators/pkg/parser"
+
+	"sigs.k8s.io/yaml"
+)
+
+// WriteToDisk writes the generated task
+// to the io.Writer
+func WriteToDisk(filename string, writer io.Writer) error {
+	file, err := os.Open(filename)
+	if err != nil {
+		return err
```

Comment:
Not sure what these errors look like in practice. Would it be clear to users what is failing, and what they would need to do to resolve?

If not, perhaps we should wrap these errors with some additional description of what was attempted.

Reply:
I use `fmt.Errorf()` to provide the description of errors.